torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -1032,9 +1075,8 @@ def perspective_bounding_box(
     #   x_out = (coeffs[0] * x + coeffs[1] * y + coeffs[2]) / (coeffs[6] * x + coeffs[7] * y + 1)
     #   y_out = (coeffs[3] * x + coeffs[4] * y + coeffs[5]) / (coeffs[6] * x + coeffs[7] * y + 1)
 
-    numer_denom_points = torch.matmul(points, theta12_T)
-    numer_points = numer_denom_points[:, :2]
```

Comment:
Hmm one last thing before reverting this. You are indexing twice here instead of splitting. The previous idiom might be faster if you were to do `tensor_split`.

Reply:
I can check that, but I think the slowdown is related to non-contig nature of splitted/indexed tensor when we try to divide one by another.

Reply:
Totally optional, nonblocking. Just an idea.